wscript

```diff
@@ -89,13 +89,15 @@ int main()
     conf.env['DEFINES_HELICS'] = ['NS3_HELICS']
 
     retval = conf.check_nonfatal(fragment=helics_test_code, lib='helics-static', libpath=conf.env['LIBPATH_HELICS'], use='HELICS')
-    print retval
+    print(retval)
     if retval:
         conf.env['HELICS'] = retval
         conf.env.append_value('LIB_HELICS', ['helics-static'])
+        conf.env.append_value('INCLUDES', conf.env['INCLUDES_HELICS'])
```

Comment:
I think this can just get moved outside the conditional since it is the same for both the debug and release HELICS libraries.

Reply:
Good catch, fixed.